### DISCUSSION TITLE
Add "Attach and attach another" to the attach modal

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#302] [Feature requests]
We could add an "Attach and attach another" button. That would not close the modal, but attach the record and let you attach another one.

![image](https://user-images.githubusercontent.com/1334409/113413443-63f77680-93c3-11eb-89ad-63267f560589.png)

**Comment by Paul-Bob** [comment]
<img width="957" height="545" alt="image" src="https://github.com/user-attachments/assets/7320839c-fe32-4fbf-814a-5bf605dc8558" />
